avm/res/network/load-balancer/tests/e2e/internal/main.test.bicep

```diff
@@ -1,5 +1,8 @@
 targetScope = 'subscription'
 
+metadata name = 'Using internal load balancer parameter'
+metadata description = 'This instance deploys the module with the minimum set of required parameters to deploy an internal load balancer.'
```

Comment:
That's not entirely true as you also deploy diagnostics etc. If you want to use this text I'd recommend removing anything that isn't needed for an internal load balancer to ensure a consumer isn't confused on what's actually required